docs/pages/billing/invoices-and-receipts.mdx

```diff
@@ -0,0 +1,137 @@
+
+For example, an [Organization account's](/accounts/account-types/#organizations) receipts are shown below:
+
+<ImageSpotlight
+  alt="Receipts page in the Expo dashboard."
+  src="/static/images/billing/receipts-01.png"
+/>
+
+### Download and view an invoice
+
+To download and view an invoice for a billing period, go to [Receipts](https://expo.dev/settings/receipts) and:
+
+- Click the **Date** for the billing period corresponding to the invoice. You will be navigated to a page hosted by Stripe. As an example, the March 22, 2024 invoice below links to this page:
+
+<ImageSpotlight
+  alt="Download invoice shown in the Stripe dashboard."
+  src="/static/images/billing/receipts-02.png"
+  style={{ maxWidth: 380 }}
+/>
+
+- Click **Download invoice**. You will receive a PDF copy of the invoice.
+
+### Download and view a receipt
+
+To download and view a receipt for a billing period, go to [Receipts](https://expo.dev/settings/receipts) and:
+
+- Click the **Date** for the billing period corresponding to the receipt. You will be navigated to the appropriate receipt hosted by Stripe. As an example, the March 22, 2024 receipt below links to this page:
+
+<ImageSpotlight
+  alt="Download receipt shown in the Stripe dashboard."
+  src="/static/images/billing/receipts-02.png"
+  style={{ maxWidth: 380 }}
+/>
+
+- Click on **Download receipt.** You will receive a PDF copy of the receipt.
+
+### Request a refund
+
+You can request a refund directly from the [Receipts](https://expo.dev/settings/receipts) page. The approval process is manual and our team investigates any errors before providing a refund.
+
+To request a refund:
+
+- Next to a receipt, click the three-dot menu and then click **Request Refund:**
+
+<ImageSpotlight
+  alt="Receipts page in the Expo dashboard."
+  src="/static/images/billing/receipts-04.png"
+/>
+
+- Fill the **Request a refund** form with details for the refund and click **Continue**:
+
+<ImageSpotlight
+  alt="Download invoice shown in the Stripe dashboard."
+  src="/static/images/billing/receipts-05.png"
+  style={{ maxWidth: 320 }}
+/>
+
+- Our billing team receives and reviews refund requests. Once a refund is approved, the amount is credited to your payment method. Refunds typically take 5 to 10 business days to fully process.
+
+## Read an invoice
+
+An invoice contains your legally registered business name, address, tax ID, invoice number, due date, and more. It also includes a description of any charges and the total amount due. In a typical invoice, the charges are divided into:
+
+- Current plan's subscription amount (if subscribed to a plan)
+- Any overage charges (if applicable)
+- A plan's credit limit
+
+Let's consider three different examples to understand how your invoice might look. If you subscribe to a [Production](/billing/plans/#production), [Enterprise](/billing/plans/#enterprise), or [On-demand](/billing/plans/#on-demand) plan, one of these scenarios may apply to you.
+
+### Subscription charges
```

Comment:
@sundeeppeswani, updated the examples as we discussed over the huddle from example invoice. Can you please take a look at the subsequent sections in the file?

The examples now matches the real-time invoices and the description now convey that message.